pymc4/distributions/mixture.py

```diff
@@ -19,11 +26,36 @@ class Mixture(Distribution):
     ----------
     p : array of floats
         p >= 0 and p <= 1
-        the mixture weights, in the form of probabilities
+        the mixture weights, in the form of probabilities, must sum to one.
     distributions : multidimensional PyMC4 distribution (e.g. `pm.Poisson(...)`)
         or iterable of one-dimensional PyMC4 distributions the
         component distributions :math:`f_1, \ldots, f_n`
     """
 
     def __init__(self, name, p, distributions, **kwargs):
         super().__init__(name, p=p, distributions=distributions, **kwargs)
+
+    @staticmethod
+    def _init_distribution(conditions, **kwargs):
+        p, d = conditions["p"], conditions["distributions"]
```

Comment:
I'm not completely sure if we should to some safety checks on `p` and `d`'s shape, but I guess that we should trust that tfp will deal with that problem for us when we create the tfd.Mixture*.